site/content/docs/user/loadbalancer.md

```diff
@@ -61,7 +61,7 @@ The output will contain a cidr such as 172.19.0.0/16.  We want our loadbalancer
 Apply the contents
 
 {{< codeFromInline lang="bash" >}}
-kubectl apply -f https://kind.sigs.k8s.io/examples/loadbalancer/metallb-configmap.yaml
+kubectl apply -f <https://kind.sigs.k8s.io/examples/loadbalancer/metallb-configmap.yaml>
```

Comment:
:)

Reply:
I swear I did not change it, must have been some vscode sorcery :(